libraries/lib-strings/Internat.cpp

```diff
@@ -135,9 +135,9 @@ bool Internat::CompatibleToDouble(const wxString& stringToConvert, double* resul
 {
    // Regardless of the locale, always respect comma _and_ point
    wxString s = stringToConvert;
-   s.Replace(wxT(","), wxString(GetDecimalSeparator()));
-   s.Replace(wxT("."), wxString(GetDecimalSeparator()));
-   return s.ToDouble(result);
+   // Convert to C locale decimal point for stable parsing.
+   s.Replace(wxString(GetDecimalSeparator()), wxT("."));
```

Comment:
Better to preserve old behavior by writing this replacement instead:
```
s.Replace(wxT(","), wxT("."))
```
So even when `GetDecimalSeparator()` is not `","`, a string with comman for decimal point still produces the same result.

This matters for certain uses of this function that are not part of serialization of files, such as the interpretation of Nyquist programs.  The old behavior in fact allows either . or , for decimal point, no matter what locale is set.  Don't change that.

Reply:
I think we should keep both replacement, like this:
```cpp
s.Replace(wxT(","), wxT("."));
s.Replace(wxString(GetDecimalSeparator()), wxT("."));
```
What you think?